README.md

```diff
@@ -67,32 +67,56 @@ git clone https://github.com/Azure-Samples/active-directory-dotnet-native-aspnet
 
 There are two projects in this sample. Each needs to be separately registered in your Azure AD tenant. To register these projects, you can:
 
-#### Navigate to the Application registration portal
+#### Choose the Azure AD tenant where you want to create your applications
+
+As a first step you'll need to:
+
+1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account or a personal Microsoft account.
+1. If your account gives you access to more than one tenant, select your account in the top right corner, and set your portal session to the desired Azure AD tenant
+   (using **Switch Directory**).
+1. In the left-hand navigation pane, select the **Azure Active Directory** service, and then select **App registrations (Preview)**.
+
+#### Register the service app (TodoListService)
+
+1. In **App registrations (Preview)** page, select **New registration**.
+1. When the **Register an application page** appears, enter your application's registration information:
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `TodoListService`.
+   - In the **Supported account types** section, select **Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com)**.
```

Comment:
Do we need to ask customer to switch to multi-tenant?

Reply:
I guess you are doing this because this app is using common, so it should be ok ;)

Reply:
I wonder also if it worth mention in the section *Step 3* that ideally user should set the app as single tenant as well.

Reply:
we could definitively have a mono-tenant app. but why would it be more ideal, @andretms. AAD v2.0 is about letting more users sign-in (MSA). It would actually be simpler for me to have a mono-tenant app, but would it demo as much? I'd like to understand your concern?

Reply:
@andretms : merging this one, but please let's discuss about having the sample illustrate more a single-tenant approach. (if we want to do that I'll send another PR)